tools/DfmHttpService/Utility.cs

```diff
@@ -56,12 +56,12 @@ public static CommandMessage GetCommandMessage(HttpListenerContext context)
             return message;
         }
 
-        public static void ReplySuccessfulResponse(HttpListenerContext context, string content, string contentType)
```

Comment:
for now, even if the response is a json type, we don't need to know its type. 

---
In reply to: [111536407](https://github.com/dotnet/docfx/pull/1547#discussion_r111536407) [](ancestors = 111536407)